src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx

```diff
@@ -312,13 +380,8 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
     this.setState({query});
   };
 
-  handleSubmit = async (
-    _data: Partial<IncidentRule>,
-    _onSubmitSuccess,
-    _onSubmitError,
-    _e,
-    model: FormModel
```

Comment:
Instead of creating a model instance for class property, can't you pass this model to `fetchStatus`?

Reply:
@billyvg I could do that but I decided that was more hacky than adding it to the class which is something we do in other places.

Reply:
imo it's less hacky because you're passing the model reference around - having it in the class to me implies that you need to access/change the model